Cargo.toml

```diff
@@ -39,7 +39,7 @@ toml_edit = { version = "0.14.4", features = ["serde"] }
 tokio = { version = "1.12.0", features = ["fs", "macros", "process", "rt-multi-thread"] }
 tracing = { version = "0.1.34", features = ["log"] }
 tracing-subscriber = "0.3.11"
-miette = { version = "4.7.1", features = ["fancy"] }
+miette = { version = "5.0.0", features = ["fancy"] }
```

Comment:
That released since the PR was published. It just adds support for treating `Arc<str>` as SourceCode, but we only use `Arc<NamedSource>`, so not particularly urgent.